### DISCUSSION TITLE
Sponsorblock, only download sponsor section

### DISCUSSION THREAD

**Discussion by matthewrhodes** [Tzahi12345/YoutubeDL-Material#1018] [Ideas]
Basicly the inversion of what it can do now, instead of cutting out sponsors and keeping the rest, it only downloads sponsors and the rest of the video is removed (or other marked section type).

Ideally, it shouldn't download the entire video, to save bandwidth.

**Comment by Tzahi12345** [comment]
Just curious what's the use case for this?
